packages/teams-js/test/private/files.spec.ts

```diff
@@ -576,15 +576,16 @@ describe('files', () => {
     it('should send the message to parent correctly', () => {
       utils.initializeWithContext('content');
 
-      const callback = jest.fn((err) => {
+      const callback = jest.fn((err, provider) => {
         expect(err).toBeFalsy();
+        expect(provider).toEqual(files.CloudStorageProvider.Dropbox);
```

Comment:
I would expect to see some unit tests for the case when an error is returned but no provider

Reply:
Added the failure test case as well.